python/ray/tune/search/nevergrad/nevergrad_search.py

```diff
@@ -120,6 +122,7 @@ def __init__(
         optimizer: Optional[
             Union[Optimizer, Type[Optimizer], ConfiguredOptimizer]
         ] = None,
+        optimizer_configs: Optional[Dict] = None,
```

Comment:
Can we actually change this back to kwargs? We use this kind of naming in other parts of Tune.
```suggestion
        optimizer_kwargs: Optional[Dict] = None,
```